nservicebus/upgrades/7to8/index.md

```diff
@@ -11,6 +11,17 @@ upgradeGuideCoreVersions:
 
 NOTE: This is a working document, as there is currently no plan or timeline for the release of NServiceBus version 8.0.
 
+## Support for external dependency injection containers
+
+Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the `NServiceBus.Extensions.DependencyInjection` library provides the ability to use any container that conforms to the `Microsoft.Extensions.DependencyInjection` container abstraction.
```

Comment:
```suggestion
Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the `NServiceBus.Extensions.DependencyInjection` package provides the ability to use any container that conforms to the `Microsoft.Extensions.DependencyInjection` container abstraction.
```

Reply:
We should add a link to the NSB.Extensions.DI documentation here.

Reply:
Yea, fixed now